### ISSUE TITLE
WakaTime Error (error in process sentinel)

### ISSUE THREAD

**Issue by prajjwol** [wakatime/wakatime-mode#12]
Hi,

I keep getting `error in process sentinel: WakaTime Error (1)`. I have verified that my plugin and the version of wakatime installed all are updated to the latest versions. Let me know if you'll need anything from me to help find the bug.

**Comment by prajjwol** [comment]
It's been solved #6

**Comment by prajjwol** [comment]
Running into this again, even after the fix.

**Comment by zm1th** [comment]
Same errors. Have tried reinstalling each piece.

Wakatime v. 4.1.8
Python 3.5.0 (homebrew)
Emacs 24.3.1
OS/X

**Comment by zm1th** [comment]
By the way, I have resolved this on my personal machine. No one mentioned that you had to also configure the wakatime cli with your api key and whatnot. I thought I just pip install, configure the emacs key value, and viola! It just works. No such luck. If you haven't tried configuring wakatime, that may be your issue.

**Comment by alanhamlett** [comment]
This should be listed in the install instructions at https://github.com/wakatime/wakatime-mode#installation

If some config/install steps are missing we can add them there.

**Comment by istinspring** [comment]
same. latest updates. i think something is broken, it happened after i updated my wakatime using pip

**Comment by oucb** [comment]
same issue when i use (setq wakatime-cli-path “/usr/local/bin/wakatime”). Try (setq wakatime-cli-path “/Library/Python/2.7/site-packages/wakatime/cli.py”), it works for me

**Comment by squiter** [comment]
This approach of set the `wakatime-cli-path` to `...wakatime/cli.py` doesn't work for me. I'm still getting the `error in process sentinel`.

**Comment by squiter** [comment]
I was upgraded the `wakatime` cli with `pip install wakatime --upgrade` and I was upgraded the `wakatime-mode` too. It didn't solve my problem, but it added a number to my error:

```
error in process sentinel: progn: WakaTime Error (103)
error in process sentinel: WakaTime Error (103)
```

**Comment by angry-bird** [comment]
@squiter  msg corresponds to error `103` is  ` An error occured while parsing ~/.wakatime.cfg`. I manually created the file `~/.wakatime.cfg` with its content: 
```
[settings]
api_key = my-api>
```
and everything is ok!

**Comment by fmeyer** [comment]
If you're getting an ```error in process sentinel: WakaTime Error (1)``` please check if you haven't installed wakatime cli with pip3. My pip was mapped to pip3, but the default python was 2.x.

**Comment by alanhamlett** [comment]
Should be fixed with 7172a92df66a69537c849182c22404715ddd9bfe and the improved initialization code.
